ServiceBusQueuePeekLockAbandon/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+    "title": "Service Bus Peek Lock Pattern - get message from a queue",
+    "description": "Get new message from a Service Bus queue and lock it for processing. The message will be deleted from the queue after the processing is completed. This pattern is useful when you want to ensure that the message is processed only once and not lost in case of a failure during processing.",
+    "prerequisites": "Before you can use this template, you need to configure a Service Bus namespace and create a queue. For more information about Service Bus see [Create a new queue on Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal)",
+    "skus": [
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Event"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        }
+    ],
+    "images": {
+        "light": "snapshot_light",
+        "dark": "snapshot_dark"
+    },
+    "parameters": [
+        {
+            "name": "queueName",
+            "displayName": "Queue Name",
+            "type": "String",
+            "default": "This is the name of the queue from which the message will be fetched.",
+            "description": "This is a parameter",
+            "required": true
+        },
+        {
+            "name": "messageBatchSize",
+            "displayName": "Message Batch Size",
+            "type": "Int",
+            "default": "50",
+            "description": "This indicates how many messages will be fetched in a single batch.",
+            "required": true
+        }
+    ],
+    "connections": {
+        "serviceBus": {
```

Comment:
Same goes for connections (key), supposed to end with _#workflowname#, this is to ensure no duplicate parameters since it's shared across workflows under the same resource.

`serviceBus_#workflowname#`

Reply:
@Elaina-Lee this should be done on both workflow and parameter definition in manifest, right?

Reply:
Updated the workflows and manifest to include _#workflowname# to parameters. @Elaina-Lee, please review. And resolve the convo if you are happy with the changes.